service-11/PUS-11-13.asn1

```diff
@@ -0,0 +1,52 @@
+--!
+--! Copyright (C) 2017 N7 Mobile sp. z o. o.
+--! Contact: http://n7mobile.pl/Space
+--!
+--! This file is part of ASN.1/ACN PUS-C Components Library.
+--!
+--! Library was developed under a programme and funded by
+--! European Space Agency.
+--!
+--! This Library is free software: you can redistribute it and/or modify
+--! it under the terms of the GNU General Public License as published by
+--! the Free Software Foundation, either version 3 of the License, or
+--! (at your option) any later version.
+--!
+--! This Library is distributed in the hope that it will be useful,
+--! but WITHOUT ANY WARRANTY; without even the implied warranty of
+--! MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--! GNU General Public License for more details.
+--!
+--! You should have received a copy of the GNU General Public License
+--! along with this program.  If not, see <http://www.gnu.org/licenses/>.
+--!
+
+PUS-11-13 DEFINITIONS AUTOMATIC TAGS ::= BEGIN
+EXPORTS ALL;
+IMPORTS
+    Subschedule-ID FROM Subschedule
```

Comment:
SubSchedule or Subschedule? just wondering :D

Reply:
Had the same problem :D. SubSchedule just looks weird to me, same as TimeBased

Reply:
how it's written in docs? if as two separate words, then SubSchedule, single word - single camel ;)

Reply:
Sub-schedule and time-based, but making an "-" looks very inconsistent when using it in the middle of message's name

Reply:
so we should go with SubSchedule - it's two words :/.